packages/http/src/validator/validators/body.ts

```diff
@@ -54,6 +55,30 @@ export function splitUriParams(target: string) {
   }, {});
 }
 
+export function parseMultipartFormDataParams(target: string, multipartBoundary?: string) {
+  if(!multipartBoundary) {
+    const error = "Boundary parameter for multipart/form-data is not defined or generated in the request header. Try removing manually defined content-type from your request header if it exists.";
```

Comment:
The boundary string MUST be non-empty if this is multipart/form-data (which it is if we're in this function). Thus, we return with error. According to https://bobbyhadz.com/blog/error-multipart-boundary-not-found-in-express-js, https://github.com/postmanlabs/postman-app-support/issues/191, https://stackoverflow.com/questions/40351429/formdata-how-to-get-or-set-boundary-in-multipart-form-data-angular, this can occur when the user is manually setting the content-type, which means the server is not generating it (and the boundary string). Thus, the possible fix is suggested in the error message.

Reply:
(Returning an error seemed to me a better solution than just returning an empty dictionary as it keeps the user more informed. Thus, we the return type is the Either monad).